kse/src/org/kse/gui/actions/resources_fr.properties

```diff
@@ -55,6 +55,11 @@ CloseOthersAction.statusbar = Fermer tous les magasins de clefs sauf celui en co
 CloseOthersAction.text      = Fermer les autres
 CloseOthersAction.tooltip   = Fermer tous les autres magasins ouverts
 
+CompareCertificateAction.statusbar             = Comparer les certificats au format ASN.1
+CompareCertificateAction.text                  = Comparer
+CompareCertificateAction.tooltip               = Comparer les certificats
+CompareCertificateAction.Title                 = Comparer les certificats
+
```

Comment:
Hi @jgrateron,

That seems good.  Then here is the French proposal:

```ini
CompareCertificateAction.onlytwo.message       = Vous ne devez s\u00E9lectionner seulement que deux certificats
```

_Similar to `DListCertificatesKS.SelectCertificate.message`._

Regards.